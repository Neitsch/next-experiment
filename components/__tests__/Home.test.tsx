import { shallow } from "enzyme";
import React from "react";

describe("Home", () => {
  let shallowRender;
  beforeEach(() => {
    jest.resetModules();
    const Home = require("../Home").default;
    shallowRender = shallow(<Home data={{ user: "Test User" }} />);
  });
  it("renders", () => {
    expect(shallowRender).toMatchSnapshot();
  });
});
