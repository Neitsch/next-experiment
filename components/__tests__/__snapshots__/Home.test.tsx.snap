// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders 1`] = `
<WithStyles(Card)>
  <WithStyles(CardContent)>
    Test User
  </WithStyles(CardContent)>
</WithStyles(Card)>
`;
